tables/geography_areas.md

```diff
@@ -1 +1,44 @@
-placeholder
+# tables/geography_areas
+
+The `geography_areas` table returns information regarding geographies within `top_level_geographies` and are identified via an `id`.
+
+The following `JOIN` queries can be carried out:
+
+- `id` on the [top_level_geographies](top_level_geographies.md) table using `top_level_geography_id`.
+- `id` on the [geography_grouping](geography_grouping.md) table using `geography_grouping_id`.
+
+## What are geography areas?
+
+A geography area relates to geography within `top_level_geographies`, for instance the geography area of `Cumbria` is located within the top-level geography of `England`.
+
+## Example use
+
+Searching within a top-level geography of `Wales`, you would find the geography area of `Gwynedd`:
+
+|id|top_level_geography_id|geography_grouping_id|description|geography_code|
+|-|-|-|-|-|
+|435|7|2,006|Gwynedd|W06000002|
+
+From this you could then continue to search for the number of people by `AGE` and `HIQUAL` (highest level of qualification) within `Gwynedd`.
+
+## Schema
+
+|column|type|use|
+|-|-|-|
+|id|int4|Primary key.|
+|top_level_geography_id|int4|Foreign key for `top_level_geographies`.|
```

Comment:
top_level_geography_id is listed as "top_level_geography" in the database, I would change it to that to avoid confusion